src/main/java/io/codelex/advancedTest/Exercise4/Test.java

```diff
@@ -0,0 +1,12 @@
+package io.codelex.advancedTest.Exercise4;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Test {
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
Ir labi, tikai šis neapgriež faila nosaukumu otrādi - bet tas sīkums, vari atstāt, kā ir!